components/language_filter/src/main.py

```diff
@@ -0,0 +1,72 @@
+"""A component that filters text based on the language."""
+import logging
+
+import fasttext
+import pandas as pd
+
+from fondant.component import PandasTransformComponent
+from fondant.logger import configure_logging
+
+configure_logging()
+logger = logging.getLogger(__name__)
+
+
+class LanguageIdentification:
```

Comment:
rather than including the ftz file, can we load from the hub since FastText is now [hosted there](https://huggingface.co/facebook/fasttext-language-identification)?

just:
```
import fasttext
from huggingface_hub import hf_hub_download

model_path = hf_hub_download(repo_id="facebook/fasttext-language-identification", filename="model.bin")
model = fasttext.load_model(model_path)
```

Reply:
What speaks against to include the ftz file in the repository? Alternative we could download the file during the image build process. Just want to avoid the situation, if some external dependencies can not be reached that the execution of the component will fail.